mapproxy/service/templates/ows_exception.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<ows:ExceptionReport xmlns:ows="http://www.opengis.net/ows/1.1"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.opengis.net/ows/1.1 http://schemas.opengis.net/ows/1.1.0/owsExceptionReport.xsd"
+  version="1.0.0" xml:lang="en">
+  <ows:Exception{{if code is not None}} exceptionCode="{{code}}"{{endif}}
+     {{if locator is not None}} locator="{{locator}}"{{endif}}>
+    <ows:ExceptionText>{{exception}}</ows:ExceptionText>
+  </ows:Exception>
+</ows:ExceptionReport>
```

Comment:
```suggestion
</ows:ExceptionReport>

```

Reply:
In this specific case, the non newline is intended :-)